neqo-transport/src/recovery.rs

```diff
@@ -20,12 +22,26 @@ use crate::flow_mgr::FlowControlRecoveryToken;
 use crate::send_stream::StreamRecoveryToken;
 use crate::tracking::{AckToken, PNSpace};
 
+const fn const_max(a: usize, b: usize) -> usize {
```

Comment:
curse you `std::cmp::{max,min}` for not being const.

Do you think we could move this to neqo-common?

Reply:
good idea.